client/packages/common/src/intl/locales/en/common.json

```
@@ -1597,5 +1597,6 @@
   "warning.cannot-create-placeholder-units": "There is a total of {{allocatedQuantity}} units available. Unable to allocate all {{requestedQuantity}} units.",
   "warning.caps-lock": "Warning: Caps lock is on",
   "warning.field-not-parsed": "{{field}} not parsed",
-  "warning.nothing-to-supply": "Nothing left to supply!"
+  "warning.nothing-to-supply": "Nothing left to supply!",
+  "messages.loading-report": "Attempting to load report..."
```

```suggestion
  "messages.loading-report": "Loading report..."
```
Mabye a simplier wording?